src/components/popover/tab/macro-tab.component.ts

```diff
@@ -42,6 +42,22 @@ export class MacroTabComponent implements OnInit, KeyActionSaver {
 
     ngOnInit() {
         this.macros = this.uhkConfigurationService.getUhkConfiguration().macros.elements;
+
+        this.macrosOptions.push({
+            id: '-1',
+            text: 'Select macro'
+        });
+
+        for (let value of this.macros) {
```

Comment:
@NejcZdovc Wouldn't this be cleaner to implement in a functional manner with `Array.map()`?